src/ducks/context/JobsContext.spec.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react'
 
 import JobsProvider, { JobsContext } from '../context/JobsContext'
-import { render } from '@testing-library/react'
-import { createMockClient } from 'cozy-client'
+import { render, act } from '@testing-library/react'
+import CozyClient from 'cozy-client'
+import CozyRealtime from 'cozy-realtime'
+
+jest.mock('cozy-realtime')
```

Comment:
You should squash the latest commit with this one. There is no need to have an intermediate version of this using CozyRealtime directly.